src/bundle/Editor.ts

```diff
@@ -509,3 +531,17 @@ export class EditorImpl extends SafeEventEmitter<EventMapInt> implements EditorI
         return serializedEditorMarkup?.trim() !== wysiwygValue.trim();
     }
 }
+
+function calculateCSSNumberValue(cssValue: string): number {
```

Comment:
Let's add a comment that this is necessary in case there is an expression in the ccc variable in order to put it in the helper in the future